cookbooks/fb_consul/files/consul.default

```diff
@@ -0,0 +1 @@
+DAEMON_ARGS='-config-file /etc/consul/consul.json'
```

Comment:
Is there other stuff that can go in here? Should we make it a template?

Reply:
yes, but it can also all go in the config file. The default does `-config-file /etc/consul/*.json` - but like with other FB cookbooks I don't want people dropping off configs outside of Chef. And since every other option can go in the config file, this is the most fb-y way to do this.

Reply:
Cool, add a comment in that case